src/routes/NavBar.svelte

```diff
@@ -1,5 +1,5 @@
 <script>
-    import { Moon, Sun } from '@steeze-ui/heroicons';
+    import { Moon, Sun, Bars3, XMark } from '@steeze-ui/heroicons';
```

Comment:
It turned out I needed to approve the workflow run.

Reply:
I tried to but I discarded the changes since a lot of other files (around 45 others) were changed. Also, if I remember correctly, sveltecheck in `pnpm fmt:fix` had an error.

Reply:
45 files changed from a `pnpm fmt:fix` run? That is interesting. @ehrelevant I'll note that @Ulyzses had a similar issue where `prettier` flagged and edited a whole bunch of files they had not modified. IIRC it was also ~40 files. 

When I ran the same command, I only had a single file modified. Someone please replicate. In the meantime, I've executed the `prettier` run myself and will be pushing the chore commit. Also, @conventoangelo could you send us a copy of a `pnpm fmt` run's logs here? 

@BastiDood you might find this interesting. Also requesting your review.

Reply:
Okay I'll take note of that.

> When I ran the same command, I only had a single file modified. Someone please replicate.
I also ran the command on the branch earlier, and can confirm that only a single file was modified for me as well. Hmmm, wonder what's causing it.

Reply:
I got it. I forgot to add `.svelte-kit/` to the `.prettierignore`. I'll go submit a PR right now to fix that. ~~I need the Hacktoberfest points.~~ 😅

Reply:
This should be addressed by #84. For your approval @VeeIsForVanana. 🚀